src/ProjectService.Mappers/RequestsMappers/Interfaces/IDbTaskMapper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.ProjectService.Mappers.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Db;
+using LT.DigitalOffice.ProjectService.Models.Dto.Requests;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Mappers.RequestsMappers.Interfaces
+{
+    [AutoInject]
+    public interface IDbTaskMapper
+    {
+        public DbTask Map(CreateTaskRequest taskRequest, Guid authorId, int taskNumber);
+    }
```

Comment:
Why DbMapper in RequestMappers?

Reply:
because we take request model for creating Db Model, as I understood your question